<launch>
  <arg name="camera_namespace" default="/camera"/>
  <arg name="color_image" default="$(arg camera_namespace)/rgb/image_rect_color"/>
  <arg name="image_out" default="image_out"/>
  <arg name="checkerboard_size_col" default="7"/>
  <arg name="checkerboard_size_row" default="5"/>
  <arg name="checkerboard_square_size" default="0.03"/>
  
  <node name="tuw_checkerboard_node" pkg="tuw_checkerboard" type="tuw_checkerboard_node" output="screen">
      <remap from="image" to="$(arg color_image)" />
      <param name="checkerboard_size_col" value="$(arg checkerboard_size_col)" type="int" />
      <param name="checkerboard_size_row" value="$(arg checkerboard_size_row)" type="int" />
      <param name="checkerboard_square_size" value="$(arg checkerboard_square_size)" type="double" />
  </node>
  <!--
  <node name="input_viewer" pkg="image_view" type="image_view">
      <remap from="image" to="$(arg color_image)" />
      <param name="autosize" value="False" />
  </node>
  <node name="output_viewer" pkg="image_view" type="image_view">
      <remap from="image" to="image_out" />
      <param name="autosize" value="False" />
  </node> -->
</launch>